hikari/impl/shard.py

```diff
@@ -709,8 +709,6 @@ async def _heartbeat_keepalive(self) -> None:
             pass
 
     async def _close_zombie(self) -> None:
-        # https://gitlab.com/nekokatt/hikari/-/issues/462
-        #
```

Comment:
FWIW so long as the GitLab repo is up this should still be valid, why should we remove links to past issues?

Reply:
The gitlab repo is archived and, to what I know, can get removed at any moment, meaning that all these links become invalid. All the comments/docstrings under this links explain exactly what the issue is, so I really don't see a point to keeping them

Reply:
The repo isn't being removed, it will stay where it is and will eventually just have pull mirroring.

It is archived atm to stop people using it to push